packages/components/Checkbox/src/Checkbox.types.ts

```diff
@@ -183,7 +188,7 @@ export interface CheckboxInfo {
 
 export interface CheckboxSlotProps {
   root: React.PropsWithRef<PressablePropsExtended>;
-  checkbox: IViewProps;
+  checkbox: React.PropsWithRef<PressablePropsExtended>;
```

Comment:
Does it need to have a ref too? I meant to just use "PressablePropsExtended"

Reply:
Changed. 
I don't we need ref too.